samcli/lib/providers/provider.py

```diff
@@ -251,7 +251,7 @@ def binary_media_types(self):
 
 _CorsTuple = namedtuple("Cors", ["allow_origin", "allow_methods", "allow_headers", "max_age"])
 
-_CorsTuple.__new__.__defaults__ = (
+_CorsTuple.__new__.__defaults__ = (  # type: ignore
```

Comment:
mypy won't acknowledge private attribute. So we need to disable it before we find an alternative